Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -1,7 +1,50 @@
+/// <reference path="shifty.d.ts" />
+const logger = new gdjs.Logger('Tween behavior');
 namespace gdjs {
-  declare var shifty: any;
-  namespace shifty {
-    export type Tweenable = any;
+  export interface RuntimeScene {
```

Comment:
Would this work with other extensions doing the same? Do you have a link to how TypeScript is handling this?

Reply:
Yes, it would. It is basically [declarations mergining](https://www.typescriptlang.org/docs/handbook/declaration-merging.html). As you can see on [TypeScript AST viewer](https://ts-ast-viewer.com/#code/MYGwhgzhAEBKDK0DeAoa0AOAnAlgNzABcBTaAfTAC5oA7AVwFsAjYraAXmgAYBuNaYAHsaEQljrBCgrAAoqtRiywBKZNAC+-MDOLVRuGgHNVqdOizFCdLDWhi6xfps0ocNElgBmYYKQTJ+Jmp6ZlYUFzcPb184RFNoMAALeR1qBFV2AD5oPEEcABNwlCA), this gets merged with the class type:
![image](https://user-images.githubusercontent.com/19349038/139951773-c850c76e-2401-4caa-a780-155e0353b79b.png)

Reply:
Ok! Resolving this comment then :)